docs/dictionary/property/HTMLText.lcdoc

```diff
@@ -28,74 +29,127 @@ write the HTMLText of field "Story" to file myWebFile
 
-The <HTMLText> <property> is a representation of the styled text of the <field(keyword)>. LiveCode uses a subset of <HTML> tags that includes font, size, style, and text color information.
+The <HTMLText> <property> is a representation of the styled text and paragraph formatting of the <field(object)>. 
+LiveCode uses a subset of <HTML> tags that includes font, size, style, and text color information, along with paragraph alignment, 
+indentation, spacing, padding and other information.
 
-Setting the <HTMLText> of a <field(keyword)> (or a <chunk> of a <field(keyword)>) sets both the text contents and the style attributes corresponding to the tags listed below. (Other tags are ignored.)
+<set(command)|Setting> the <HTMLText> of a <field(object)> (or a <chunk> of a <field(object)>) sets both the text contents and 
+the style attributes corresponding to the tags listed below. (Other tags are ignored.)
 
-Getting the <HTMLText> <property> reports a <string> consisting of the text of the <field(keyword)> (or <chunk> of a <field(keyword)>), with any font, size, style, or color <properties> embedded in the text in the form of the tags listed below.
+<get(command)|Getting> the <HTMLText> <property> reports a <string> consisting of the text of the <field(object)> (or <chunk> of 
+a <field(object)>), with any font, size, style, or color <properties> embedded in the text in the form of the tags listed below.
 
 The tags translate as follows:
 
 &lt;p&gt; &lt;/p&gt;
-Encloses a line of text. (Blank lines are also enclosed in &lt;p&gt; &lt;/p&gt;.)
+Encloses a line of text. (Blank lines are also enclosed in &lt;p&gt; &lt;/p&gt;.) 
+These properties are represented as attributes of the &lt;p&gt; tag:
+* align="left|center|right" appears if an <textAlign|alignment> has been applied to the line.
+* firstindent="N" (where N is an integer representing number of pixels) appears if the <firstIndent> has been set for the line.
+* leftindent="N" appears if the <leftIndent> has been set for the line.
+* rightindent="N" appears if the <rightIndent> has been set for the line.
+* spaceabove="N" appears if the <spaceAbove> has been set for the line.
+* spacebelow="N" appears if the <spaceBelow> has been set for the line.
+* tabstops="N" appears if <tabStops> have been set for the line.
+* borderwidth="N" if a <borderWidth> has been set for the line.
+* hgrid="true|false" if the <vGrid> property has been set for the line.
+* vgrid="true|false" if the <hGrid> property has been set for the line.
+* dontwrap="true|false" if the <dontWrap> property has been set for the line.
+* padding="N" if the <padding> has been set for the line.
+* hidden="true|false" if the <hidden> property has been set for the line.
+* bgcolor="#NNNNNN" if a <backgroundColor> has been set for the line.
+* bordercolor="#NNNNNN" if a <borderColor> has been set for the line.
+>*Note*: An <HTML>-style color definition consists of a hash mark (#) followed by three 2-digit hexadecimal numbers, 
```

Comment:
Existing Note blockquotes have the colon inside the bold section rather than outside.